packages/gafl-webapp-service/src/pages/buy-or-renew/__tests__/update-transaction.spec.js

```diff
@@ -28,15 +24,15 @@ describe('buy-or-renew > update-transaction', () => {
   describe('default', () => {
     it('should set buyNewLicence to true on the transaction cache, if user has selected buy-licence', async () => {
       mockPageCacheGet.mockImplementationOnce(() => ({ payload: { 'buy-or-renew': 'buy-licence' } }))
-      mockTransactionCacheGet.mockImplementationOnce(() => ({}))
+      mockStatusCacheGet.mockImplementationOnce(() => ({}))
 
       await updateTransaction(mockRequest)
       expect(mockStatusCacheSet).toHaveBeenCalledWith({ buyNewLicence: true })
     })
 
     it('should set buyNewLicence to false on the transaction cache, if user has selected renew-licence', async () => {
       mockPageCacheGet.mockImplementationOnce(() => ({ payload: { 'buy-or-renew': 'renew-licence' } }))
-      mockTransactionCacheGet.mockImplementationOnce(() => ({}))
+      mockStatusCacheGet.mockImplementationOnce(() => ({}))
```

Comment:
Just had a think about this - rather than doing `mockImplementationOnce`, it'd be better to have this in the jest.fn on line 8, then lines 27 and 35 can be eliminated. This is a pattern used in a lot of places and I think we should get rid of it as far as possible as it complicates the code.